src/run.rs

```diff
@@ -86,8 +86,7 @@ pub fn run() {
          .short("f")
          .long("justfile")
          .takes_value(true)
-         .help("Use <JUSTFILE> as justfile. --working-directory must also be set")
-         .requires("WORKING-DIRECTORY"))
+         .help("Use <JUSTFILE> as justfile. --working-directory can be set [default: supplied justfile location]"))
```

Comment:
I think that since the behavior of just in setting the working directory is the same as normal when using this flag, we can make the comment shorter. Just `Use <JUSTFILE> as justfile.` is fine.

Reply:
Done.